anime_downloader/cli.py

```diff
@@ -30,6 +30,10 @@ def cli():
 @click.option(
     '--episodes', '-e', 'episode_range', metavar='<int>:<int>',
     help="Range of anime you want to download in the form <start>:<end>")
+@click.option(
```

Comment:
I think we don't need a separate CLI argument for this. This functionality should be added to `--episodes-range` (It is not ranges anymore then, but I think we can ignore it). 

`util.split_anime` can be changed to facilitate this functionality. Maybe it's name should be changed to `parse_ep_str` (or something).

Reply:
Indeed, I can change the name of the parse_grammar function to parse_episode_string. I would personally like to keep split_anime the way it is and have the new episode parser use it like it is now. I'll drop the option from the command line and place the new episode parser in.